src/components/AboutContent.vue

```diff
@@ -63,7 +59,7 @@
           >
             our documentation</a
           >
-          are shown below.
+          are shown below (in an actively randomised order).
```

Comment:
I would definitely remove that parenthesis comment. People don't (shouldn't have to) care about this.

Reply:
For me it is important so visitors don't come to any preconceived notion on importance due to the order of the names.

Reply:
Are there not more important things in life?